src/main.py

```diff
@@ -181,12 +187,13 @@ async def on_message(message: discord.Message):
             await message.channel.send(lvl_up_message)
 
     for log_link in re.findall(r"https://smapi.io/log/[a-zA-Z0-9]{32}", message.content):
-        log_dict = requests.get(f"http://api.pil.ninja/smapi_log/endpoint?{log_link}").json()
-        if log_dict["success"]:
-            windows_info = re.search(r"(Windows (?:Vista|\d+)) .+", log_dict["OS"])
-            if windows_info:  # Condense OS text for Windows because it's often quite verbose.
-                log_dict["OS"] = windows_info.group(1)
-            await message.channel.send(smapi_log_message_template.substitute(log_dict))
+        log_info = parse_log(log_link)
+        
+        if log_info["suggested_fixes"] != []:
```

Comment:
I think this needs to check for a empty string now.

Reply:
True